ui/dashboards/src/components/Variables/VariableEditorForm.tsx

```diff
@@ -0,0 +1,306 @@
+// Copyright 2022 The Perses Authors
```

Comment:
Just a general observation as I'm starting to look at this component: this file is getting pretty big. Do you think there's an opportunity to break it up a bit into sub-components or separate out some of the "model" stuff into its own file(s)?

Reply:
Yeah good call, I'll pull out the functions